base/README.md

```diff
@@ -9,5 +9,6 @@ https://registry.hub.docker.com/u/ceph/base/
 ## Usage (example)
 
 ```bash
-$ docker run -i -t ceph/base /bin/bash
+$ docker run --rm -ti ceph/base
```

Comment:
I don't think we should default to `--rm` in the example for base.  If people are using base directly, they are probably trying to construct their own images for use.